### PR TITLE
tox.ini: clean up changedir

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -95,13 +92,12 @@ deps =
...
     {env:_PYTEST_TOX_EXTRA_DEP:}
 distribute = true
-changedir=testing
 setenv =
     {[testenv]setenv}
     PYTHONDONTWRITEBYTECODE=1
 passenv = USER USERNAME TRAVIS
 commands =
-    {env:_PYTEST_TOX_COVERAGE_RUN:} pytest -n auto {posargs:.}
+    {env:_PYTEST_TOX_COVERAGE_RUN:} pytest -n auto
```
Comment (nicoddemus) [inline tox.ini]: Why are you removing `posargs` from some testenvs? I find them useful to pass extra arguments to envs.

Comment (blueyed) [reply tox.ini]: Yes, agreed - this was not intentional.

Comment (blueyed) [reply tox.ini]: It was just one though, fixed.

**Comment by nicoddemus** [comment]
Flaky failure: #4144

**Pull request merged**